src/renderer/features/governance/aggregates/delegatedVotes.ts

```diff
@@ -0,0 +1,49 @@
+import { combine, sample } from 'effector';
+import uniq from 'lodash/uniq';
+
+import { nonNullable, toAddress } from '@/shared/lib/utils';
+import { referendumModel } from '@/entities/governance';
+import { walletModel } from '@/entities/wallet';
+import { delegatedVotesModel } from '../model/delegatedVotes';
+import { networkSelectorModel } from '../model/networkSelector';
+
+import { proposerIdentityAggregate } from './proposerIdentity';
+
+const $delegatedVotesInChain = combine(
+  delegatedVotesModel.$delegatedVotes,
+  networkSelectorModel.$governanceChain,
+  (votes, chain) => {
+    if (!chain) return {};
+
+    return votes[chain.chainId] ?? {};
+  },
+);
+
+sample({
+  clock: [referendumModel.events.requestDone, networkSelectorModel.$governanceChainApi, walletModel.$activeWallet],
```

Comment:
is it possible for this clock to be triggered if we have `networkSelectorModel.$governanceChainApi`, but already left the gov page and switching between wallets?

Reply:
delegated votes are separated by chains, so it's same behavior as anywhere in gov(